packages/fxa-settings/src/components/UnitRowTwoStepAuth/index.tsx

```diff
@@ -54,10 +61,13 @@ export const UnitRowTwoStepAuth = () => {
     ? {
         headerValueClassName: 'text-green-800',
         headerValue: 'Enabled',
-        ctaText: 'Change',
         secondaryCtaText: 'Disable',
         secondaryButtonClassName: 'cta-caution',
+        // TODO(DJ): The naming of this is a bit confusing, since they are swapped in this
+        // case, we should come up with a better name here.
```

Comment:
Was this intended to be fixed in this PR? If not, would you mind filing a new ticket and sticking the FXA-XXX here?